quartical/apps/backup.py

```diff
@@ -41,7 +42,14 @@ def backup():
         args.ms_path,
         columns=args.column,
         index_cols=("TIME",),
-        group_cols=("DATA_DESC_ID",))
+        group_cols=("FIELD_ID","DATA_DESC_ID","SCAN_NUMBER"))
+
+
+    for i, ds in enumerate(data_xds_list):
+        chunks = {'row':125000}
+        if 'chan' in ds.dims:
+            chunks['chan'] = 128
```

Comment:
Adapt based on ncorr? Also, largest dtype in the MS is actually `np.complex64`. Maybe:

```python
for i, ds in enumerate(data_xds_list):
    chunks = {'row':125000}
    if 'chan' in ds.dims:
         chan_chunk = 256 / (ds.dims['corr'] if 'corr' in ds.dims else 1) 
         chunks['chan'] = chan_chunk
```

Reply:
Actually, given that we are only ever doing this one column at a time, we can adapt both on dtype and dims. Should if you would like me to take over @landmanbester i.e. if this is getting more annoying than anticipated.

Reply:
I have no issue with you taking over. I would probably do this in an if/else ladder and I know how much you hate those ;-)